packages/webdriverio/src/scripts/isDisplayedInViewport.js

```diff
@@ -1,12 +1,8 @@
-
 /**
  * check if element is visible and within the viewport
- *
- * @param {String}  elements  DOM elements to check against
- *
- * @see  waitForVisible
+ * @param  {HTMLElement} elem  element to check
+ * @return {Boolean}           true if element is within viewport
  */
-
 export default function isDisplayedInViewport (elem) {
```

Comment:
Why go from visible to displayed? All methods in the webdriverio api use visible, I suggest to keep this consistent.
So my proposal is to either change isVisible to isDisplayed and such, or we change this method back to isVisibleInViewport.

Reply:
@erwinheitzman we changed that command from isVisible to isDisplayed (https://github.com/webdriverio/v5/blob/master/packages/webdriverio/src/commands/element/isDisplayed.js)